ui/components/ui/icon/copy-contract-details.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CopyContractDetails = ({ className }) => (
+  <svg
+    className={className}
+    width="18"
+    height="18"
+    viewBox="0 0 18 18"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M12.3333 9.75008V13.2501C12.3333 16.1667 11.1667 17.3334 8.24999 17.3334H4.74999C1.83332 17.3334 0.666656 16.1667 0.666656 13.2501V9.75008C0.666656 6.83341 1.83332 5.66675 4.74999 5.66675H8.24999C11.1667 5.66675 12.3333 6.83341 12.3333 9.75008Z"
+      fill="#BBC0C5"
```

Comment:
@georgewrmarshall Do we have a `var()` for this?

Reply:
ooh, can we pass --vars as html attributes for these like in the case of fill?